vaadin-grid-pro-flow-integration-tests/src/main/java/com/vaadin/flow/component/gridpro/examples/MainView.java

```diff
@@ -36,6 +39,14 @@ protected void createEditorColumns() {
             subPropertyDisplayPanel.setText(newValue);
         }).setHeader("Name").setWidth("300px");
 
+        ComboBox<Department> cb = new ComboBox<>();
+        cb.setItems(Department.values());
+        grid.addEditColumn(Person::getDepartment).custom(cb, (item, newValue) -> {
+            item.setDepartment(newValue);
+            itemDisplayPanel.setText(item.toString());
+            subPropertyDisplayPanel.setText(newValue.toString());
```

Comment:
Throws null pointer on empty selection. For that reason it's usually better to use `String.valueOf(newValue)`

Reply:
Updated.